tests/cxx/include_with_using.cc

```diff
@@ -41,7 +41,6 @@ void TestDeclFromD4() {
   var_in_d4 = 0;
 }
 
-// IWYU: i1::UsingInCc needs a declaration
```

Comment:
The reason is because we don't actually 'use' anything here at this point anymore, it's completely deferred to the use point.  Previously, we would add all of the shadow decls when coming across a using decl, but now we don't so there's nothing to issue a warning against.  I think this should be fine because the warning will simple occur at the point of use instead of here.